parm/config/config.resources

```diff
@@ -188,6 +188,24 @@ elif [ ${step} = "atmanalpost" ]; then
 
 elif [ ${step} = "aeroanlinit" ]; then
 
+   case ${CASE} in
+     C768)    
+        layout_x=6
```

Comment:
Are the C768 and C384 `layout_x` and `layout_y` placeholders?   If I correctly understand how these values are used, it's odd to specify smaller values for higher resolutions.

Reply:
Yes they are placeholders, I was using fewer PEs for the lower resolutions but because BUMP requires PEs_BUMP == PEs_Var, we increased it because @andytangborn was using 384 PEs to compute his initial **B**